elisp/intero.el

```diff
@@ -2001,7 +2003,7 @@ config exists."
                                             "--project-root"
                                             "--verbosity" "silent"))
               (0 (buffer-substring (line-beginning-position) (line-end-position)))
-              (t (intero--warn "Couldn't get the Stack project root.
+              (t (intero--warn (format "Couldn't get the Stack project root.
```

Comment:
You don't need the `format` here: see the docstring & signature for `intero--warn`.